ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -843,6 +875,24 @@ public class BottomSheetController: UIViewController {
         return maxHeight
     }
 
+    // Output of `collapsedHeightResolver` wrapped in a cache.
+    private var resolvedCollapsedSheetHeight: CGFloat {
+        let oldContext = lastCollapsedSheetHeightResolutionContext
+        let newContext = ContentHeightResolutionContext(maximumHeight: maxSheetHeight - view.safeAreaInsets.bottom, containerTraitCollection: view.traitCollection)
+
+        if oldContext?.maximumHeight != newContext.maximumHeight || !(oldContext?.containerTraitCollection.containsTraits(in: newContext.containerTraitCollection) ?? false) {
```

Comment:
Maybe I'm just weak, but optional chaining combined with a boolean operator (`||`) really confuses me. This seems like a good case for a
```
if let oldContext = oldContext,
oldContext.maximumHeight != newContext.maximumHeight || !oldContext.containerTraitCollection.containsTraits(in: newContext.containerTraitCollection) {
}
```